digitalfabrik/entitlementcard
# 1624: Add role for verein360 to create api tokens

### Short description
Added a new role for verein360 admins

### Proposed changes
- added a new role to create api tokens for verein360
- added new column "type" for api tokens
- adjusted project settings
- new external-user role can only create tokens of type "verified_application"
- project admins of koblenz create tokens of type "user_import"
- only api tokens of type user-import are allowed to use koblenz csv endpoint

### Side effects
- may break anything releated to api token generation of koblenz project admins or csv upload endpoint

### Testing
- create and delete tokens for different roles, check if type is set correctly
- check if endpoint for koblenz csv shows forbidden exception when used with api token of type external-verified
- add and remove user with external-api role

### Resolved issues



Fixes: #1624
